### PR TITLE
Fix bug about switching flight control mode by xyControlModeCallback

### PR DESCRIPTION
**Pull request by kanke-hayata**

### CODE REVIEW

File: aerial_robot_control/include/aerial_robot_control/flight_navigation.h [hunk 1]
```diff
@@ -441,11 +441,15 @@ namespace aerial_robot_navigation
         {
           if(msg->data == 0)
             {
+              setTargetXyFromCurrentState();
+              target_acc_.setValue(0, 0, 0);
```
Comment (chibi314) [inline aerial_robot_control/include/aerial_robot_control/flight_navigation.h:445]: `target_vel_` should be also reset here?

Comment (tongtybj) [reply aerial_robot_control/include/aerial_robot_control/flight_navigation.h:445]: @kanke-hayata 

I also agree with @chibi314 . Please add `target_vel_.setValue(0, 0, 0);` here

**Review by tongtybj — COMMENTED** [review]
Please consider @chibi314 comment

File: aerial_robot_control/include/aerial_robot_control/flight_navigation.h [hunk 2]
```diff
@@ -441,11 +441,16 @@ namespace aerial_robot_navigation
         {
           if(msg->data == 0)
             {
+              setTargetXyFromCurrentState();
+              target_vel_.setValue(0, 0, 0);
```
Comment (kanke-hayata) [inline aerial_robot_control/include/aerial_robot_control/flight_navigation.h:445]: I added this line.

**Review by tongtybj — APPROVED** [review]

**Review by chibi314 — APPROVED** [review]

**Pull request merged**